### ISSUE TITLE
Review and refine user documentation

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#2498]
Find out what is missing? There are several open issues regarding documentation:
https://github.com/dzhw/metadatamanagement/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+document

**Comment by rbirkelbach** [comment]
https://github.com/dzhw/metadatamanagement/issues/2502
https://github.com/dzhw/metadatamanagement/issues/2292
In my opinion this should be done first (especially because of nacaps): https://github.com/dzhw/metadatamanagement/issues/2266
https://github.com/dzhw/metadatamanagement/issues/1417

**Comment by svietgen** [comment]
@rbirkelbach I just noticed that this can be deleated https://metadatamanagement.readthedocs.io/de/latest/excel_templates.html

**Comment by svietgen** [comment]
@rbirkelbach This can be deleated too (I guess) Or it needs a review:
https://metadatamanagement.readthedocs.io/de/latest/stata_skripte.html

**Comment by rbirkelbach** [comment]
> @rbirkelbach This can be deleated too (I guess) Or it needs a review:
> https://metadatamanagement.readthedocs.io/de/latest/stata_skripte.html

Thank you. I created a new issue #2541 for improving documentation for rdc-employees/publishers, as it is slightly out of scope here. Issue #2498 is for public users/data providers.
